node-wasm/src/lib.rs

```diff
@@ -1,8 +1,13 @@
 #![doc = include_str!("../README.md")]
 #![cfg(target_arch = "wasm32")]
 
+pub mod client;
+mod commands;
 pub mod error;
-pub mod node;
+mod ports;
 pub mod utils;
 mod worker;
 mod wrapper;
+
+#[cfg(all(target_arch = "wasm32", test))]
```

Comment:
this crate is completely empty in non-wasm anyway :p
```suggestion
#[cfg(test)]
```